src/printer.js

```diff
@@ -1907,19 +1907,17 @@ function genericPrintNoParens(path, options, print, args) {
           // quasi literal), therefore we want to indent the JavaScript
           // expression inside at the beginning of ${ instead of the beginning
           // of the `.
-          let size = 0;
-          const value = childPath.getValue().value.raw;
-          const index = value.lastIndexOf("\n");
           const tabWidth = options.tabWidth;
-          if (index !== -1) {
-            size = util.getAlignmentSize(
-              // All the leading whitespaces
-              value.slice(index + 1).match(/^[ \t]*/)[0],
-              tabWidth
-            );
-          }
+          const indentSize = util.getIndentSize(
+            childPath.getValue().value.raw,
+            tabWidth
+          );
```

Comment:
Is this change needed for this PR? What does it do?

Reply:
@lydell I had refactored out `getIndentSize` because I was using it previously with `align`, but I can revert it since it's no longer needed (although I don't think it hurts to leave it factored out into its own function).

Reply:
I see. It’s a nice cleanup so we can keep it refactored out. 👍